packages/user-menu/src/lib/user-menu/components/user-menu/user-menu.component.scss

```diff
@@ -0,0 +1,23 @@
+@import '~@a-ui/core/dist/assets/styles/_quarks';
+
+.a-button-avatar {
```

Comment:
A few remarks:

- there are some indenting issues
- please add spaces above new selectors
- make sure all colons are followed by a space
- please arrange your properties in alphabetical order (we try to do this in all projects)